src/crypto/tls/common.go

```diff
@@ -1135,9 +1178,14 @@ func supportedVersionsFromMax(maxVersion uint16) []uint16 {
 }
 
 var defaultCurvePreferences = []CurveID{X25519, CurveP256, CurveP384, CurveP521}
+var defaultKEMPreferences = []CurveID{SIKEp434, Kyber512, X25519, CurveP256, CurveP384, CurveP521}
```

Comment:
```suggestion
var defaultKEMTLSPreferences = []CurveID{SIKEp434, Kyber512, X25519, CurveP256, CurveP384, CurveP521}
```
This will make it easier to find this variable.

Also, I'd remove the NIST curves unless we plan to use them.

Reply:
The NIST curves are the fallback algorithms; so yes, they could be used if you support some kem but the peer did not. 

> unless we plan to use them

what does this mean?

Reply:
Hmm, in retrospect I think this comment didn't make much sense. What I meant was: "KEMTLS clients don't need to advertise support for these curves". Looking at the code closer, I realized that the may end up falling back to these curves if the server doesn't support KEMTLS.

Disregard the comment about the NIST curves. I still think we should change the name, however.